ui/src/components/Bouts/FeedItem.tsx

```diff
@@ -0,0 +1,370 @@
+          alignSelf={"flex-start"}
+        >
+          <Box>
+            <Typography component="div" mr={3} variant="overline">
+              Detections
+            </Typography>
+            <Box
+              alignItems="center"
+              display="flex"
+              justifyContent="space-between"
+            >
+              <Typography component="div" mr={3}>
+                {detsCount5MinAgo}{" "}
+                <Typography
+                  fontSize="small"
+                  noWrap
+                  fontWeight="bold"
+                  color={theme.palette.accent2.main}
+                >
+                  5 min
+                </Typography>
+              </Typography>
+              <Typography component="div" mr={3}>
+                {detsCount15MinAgo}
+                <Typography
+                  fontSize="small"
+                  noWrap
+                  color={theme.palette.accent2.main}
+                  fontWeight="bold"
+                >
+                  15 min
+                </Typography>
+              </Typography>
+              <Typography component="div" mr={5}>
+                {detsCount}{" "}
+                <Typography
+                  fontSize="small"
+                  noWrap
+                  color={theme.palette.accent2.main}
+                  fontWeight="bold"
+                >
+                  1 hr
+                </Typography>
+              </Typography>
+            </Box>
+          </Box>
+
+          <Box sx={{ my: { xs: 2, sm: 0 } }}>
+            <Typography component="div" mr={3} variant="overline">
+              Categories
+            </Typography>
+            <Box
+              alignItems="center"
+              display="flex"
+              justifyContent="space-between"
+            >
+              {categories.map((cat, i) => (
+                <Typography key={i} component="div" mr={3}>
+                  {
+                    recentDetections.filter(({ category }) => cat === category)
+                      .length
+                  }{" "}
+                  <Typography
+                    fontSize="small"
+                    noWrap
+                    color={theme.palette.accent2.main}
+                    fontWeight="bold"
+                  >
+                    {cat}
+                  </Typography>
+                </Typography>
+              ))}
+            </Box>
+          </Box>
+
+          <Box alignSelf={"stretch"} sx={{ my: { xs: 2, sm: 0 } }}>
+            <Typography component="div" mr={3} variant="overline">
+              Listeners
+            </Typography>
+            <Typography display="flex" alignItems="center" fontSize={14}>
+              <Person sx={{ mr: 1, color: theme.palette.accent2.main }} />
+              {listenerCount === undefined ? "-" : listenerCount}
+            </Typography>
+          </Box>
+        </Box>
+      </Box>
+
+      {/* Detections charts */}
```

Comment:
I feel like if you have to comment the section, it should be its own component

Reply:
Let's extract it once the UI stabilizes